src/Compilers/VisualBasic/Portable/CommandLine/VisualBasicCommandLineParser.vb

```diff
@@ -399,6 +400,13 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
 
                 If IsScriptRunner Then
                     Select Case name
+                        Case "i", "i+"
+                            interactiveMode = True
```

Comment:
For consistency with the C# version, insert "If value IsNot Nothing Then Exit Select" into both new cases.